<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMainPresenter</class>
 <widget class="QMainWindow" name="AdminMainPresenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>771</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QTableWidget" name="LibraryTable"/>
     </item>
     <item row="0" column="1">
      <widget class="QTableWidget" name="BooksTable"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>681</width>
      <height>236</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="2" column="3">
      <widget class="QLineEdit" name="BookNameText_2"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LibraryIDText"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>library name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="BookIDText"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="AddressText_2"/>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="UpdateLibraryButton">
       <property name="text">
        <string>update library</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="LibraryNameText_2"/>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="DeleteLibraryButton">
       <property name="text">
        <string>delete library</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>library address</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QPushButton" name="UpdateBookButton">
       <property name="text">
        <string>update book</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>book author</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>library id</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="2">
      <widget class="QPushButton" name="AddBookButton">
       <property name="text">
        <string>add book</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>book id</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="AuthorText_2"/>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QPushButton" name="DeleteBookButton">
       <property name="text">
        <string>delete book</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="AddLibraryButton">
       <property name="text">
        <string>add library</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>book name</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
